cypress/support/commands.js

```diff
@@ -93,3 +96,23 @@ Cypress.Commands.add("restartWiremock", (stubDir) => {
   cy.task("stopWiremock");
   cy.task("startWiremock", { stubDir });
 });
+
+Cypress.Commands.add("selectFacility", () => {
+  cy.get("body").then(($body) => {
+    if (
+      $body
+        .text()
+        .includes(
+          "Please select the testing facility where you are working today."
+        )
+    ) {
+      cy.get(".usa-card__body").last().click();
+    }
```

Comment:
Nice workaround! This should help the e2e tests behave more like they're idempotent, even though they're really not (when run locally, at least).